docs/tutorials/integrations/kserve/kserve-custommetrics-one.md

```diff
@@ -0,0 +1,117 @@
+kubectl port-forward --namespace istio-system svc/$INGRESS_GATEWAY 8080:80
+```
+
+Finally, request predictions. Here we make a request about once a second.
+
+```shell
+SERVICE_HOSTNAME="sklearn-iris.default.example.com"
+# kubectl get inferenceservice sklearn-iris -o jsonpath='{.status.url}' | cut -d "/" -f 3
+
+
+cat <<EOF > "./iris-input.json"
+{
+  "instances": [
+    [6.8,  2.8,  4.8,  1.4],
+    [6.0,  3.4,  4.5,  1.6]
+  ]
+}
+EOF
+
+while true; do 
+  curl -H "Host: ${SERVICE_HOSTNAME}" \
+    http://localhost:8080/v1/models/sklearn-iris:predict \
+    -d @./iris-input.json
+  sleep 1
+done
+```
+
+## Launch Iter8 Experiment
+
+Launch the Iter8 experiment inside the Kubernetes cluster:
+
+```shell
+iter8 k launch \
+--set "tasks={ready,custommetrics,assess}" \
+--set ready.isvc=sklearn-iris \
+--set ready.timeout=180s \
+--set custommetrics.templates.kserve-prometheus="https://gist.githubusercontent.com/kalantar/adc6c9b0efe483c00b8f0c20605ac36c/raw/c4562e87b7ac0652b0e46f8f494d024307bff7a1/kserve-prometheus.tpl" \
+--set custommetrics.values.labels.service_name=sklearn-iris-predictor-default \
+--set "custommetrics.values.latencyPercentiles={50,75,90,95}" \
+--set assess.SLOs.upper.kserve-prometheus/error-count=0 \
+--set assess.SLOs.upper.kserve-prometheus/latency-mean=25 \
+--set assess.SLOs.upper.kserve-prometheus/latency-p'90'=40 \
+--set runner=cronjob \
+--set cronjobSchedule="*/1 * * * *"
+```
+
+??? note "About this experiment"
+    This experiment consists of three [tasks](../../../getting-started/concepts.md#iter8-experiment), namely, [ready](../../../user-guide/tasks/ready.md), [custommetrics](../../../user-guide/tasks/custommetrics.md) and [assess](../../../user-guide/tasks/assess.md). 
+
+    The [ready](../../../user-guide/tasks/ready.md) task checks if the `sklearn-iris` InferenceService exists and is `Ready`. 
+
+    The [custommetrics](../../../user-guide/tasks/custommetrics.md) task checks reads metrics from a Prometheus service as defined by the template.  
+
+    The [assess](../../../user-guide/tasks/assess.md) task verifies if the model satisfies the specified SLOs:
+
+    - there are no errors
+    - the mean latency of the prediction does not exceed 25 msec, and
+    - the 90th percentile latency for prediction does not exceed 40 msec. 
+    
+    This is a [multi-loop](../../../getting-started/concepts.md#iter8-experiment) [Kubernetes experiment](../../../getting-started/concepts.md#kubernetes-experiments); its [runner](../../../getting-started/concepts.md#runners) is `cronjob`. The `cronjobSchedule` expression specifies the frequency of the experiment execution -- periodically refreshing the metric values and performing SLO validation using the updated values.
+
+***
+
+You can assert experiment outcomes, view an experiment report, and view experiment logs as described in [your first experiment](../../../getting-started/your-first-experiment.md).
+
+## Clean up
+
+```shell
```

Comment:
Cleanup usually provides a description of what is being cleaned up (see http://localhost:8000/latest/tutorials/load-test-grpc/)... please provide a description of what is getting cleaned up like in the gRPC example.
***
Also, what about cleaning up Prometheus and KServe? Prometheus removal is [here](https://github.com/kserve/kserve/tree/master/docs/samples/metrics-and-monitoring#removal).

Reply:
I could not find documentation for removing KServe. I don't think we want to write our own. This involves Istio, KNative, Cert manage and KServe.  We could describe just removal of KServe but we give quick install link that installs everything.